server-test/app/page.tsx

```diff
@@ -54,10 +70,13 @@ Markdown *emphasis* word (expectedly not translated to italic).
 </p>
 `;
 
+const pureHtmlText = `
+<p class="PlaygroundEditorTheme__paragraph" dir="ltr"><span style="white-space: pre-wrap;">Greg’s career began in manufacturing. </span></p><p class="PlaygroundEditorTheme__paragraph" dir="ltr"><br></p><p class="PlaygroundEditorTheme__paragraph" dir="ltr"><u><span class="PlaygroundEditorTheme__textUnderline" style="white-space: pre-wrap;">{{Tenant_First_Name}}{{Tenant_Last_Name}} </span></u></p><p class="PlaygroundEditorTheme__paragraph" dir="ltr"><br></p><p class="PlaygroundEditorTheme__paragraph" dir="ltr"><span style="white-space: pre-wrap;">The </span><b><strong class="PlaygroundEditorTheme__textBold" style="white-space: pre-wrap;">company</strong></b><span style="white-space: pre-wrap;"> he started with focused on using lean manufacturing techniques. Lean manufacturing focuses on eliminating waste and improving process flow when building a product. </span></p><p class="PlaygroundEditorTheme__paragraph" dir="ltr"><br></p><p class="PlaygroundEditorTheme__paragraph" dir="ltr"><span style="white-space: pre-wrap;">One of the people who influenced the creation of lean manufacturing was W. Edwards Deming, a noted statistician and manufacturing consultant. In his book Out of the Crisis, Deming out- lines 14 points for management. One of those points appears at the beginning of this chapter and has resonated with Greg his entire career. Building quality into the product sounds clichéd and has been overused by many marketing departments. But in an agile environment, the concept is real and tangi-ble. Consider the following. {{Tenant_ID_No}} {{Tenant_Email}}</span></p>`;
```

Comment:
The library is processing the tag `<b>` but the style seems to be in the `<strong>` tag. Am I wrong?

Reply:
the b tag is wrapped around the strong tag when the html is generated from the lease template text.